lua/gk.lua

```diff
@@ -1,16 +1,46 @@
 local gatekeeperc = require("gatekeeperc")
+local ffi = require("ffi")
 
 local M = {}
 
+function init_tunnel(conf, gw_addr, gk_addrs, gt_addrs)
+	local gk_strs = ffi.new("const char *[" .. #gk_addrs .. "]")
+	for i, v in ipairs(gk_addrs) do
+		gk_strs[i - 1] = v
+	end
+
+	local gt_strs = ffi.new("const char *[" .. #gt_addrs .. "]")
+	for i, v in ipairs(gt_addrs) do
+		gt_strs[i - 1] = v
+	end
+
+	return gatekeeperc.lua_init_tunnel(conf, gw_addr,
+		gk_strs, #gk_addrs,
+		gt_strs, #gt_addrs)
+end
+
 -- Function that sets up the GK functional block.
 function M.setup_block(net_conf)
 
 	-- Init the GK configuration structure.
 	local conf = gatekeeperc.alloc_gk_conf()
+	
+	-- Change these parameters to configure the Gatekeeper.
+	local gw_addr  = "e8:ea:6a:06:21:b3"
```

Comment:
As mentioned in previous comments, perhaps this Ethernet address could be taken from the net configuration as the Ethernet address of the back interface.